### PR TITLE
Run pre-commit hooks

### PR DESCRIPTION
**Pull request by lunaticare**
I could not create a commit locally with [pre-commit hook installed](https://github.com/haskell/haskell-language-server/blob/master/docs/contributing/contributing.md#formatter-pre-commit-hook). Ran it locally on the clean branch:

```sh
pre-commit run --all-files
```

<a href="https://gitpod.io/#https://github.com/haskell/haskell-language-server/pull/3059"></a>

### CODE REVIEW

**Review by Ailrun — APPROVED** [review]
LGTM. Thank you :)

File: plugins/hls-brittany-plugin/test/testdata/BrittanyCRLF.formatted_range.hs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-foo ::   Int   -> String-> IO ()
+foo ::   Int   -> String-> IO ()
```
Comment (Ailrun) [inline plugins/hls-brittany-plugin/test/testdata/BrittanyCRLF.formatted_range.hs:1]: Oops, this should not be formatted. Could you add this to the ignore file list?

**Review by Ailrun — CHANGES_REQUESTED** [review]

File: plugins/hls-brittany-plugin/test/testdata/BrittanyCRLF.formatted_range.hs (hunk 1 above)
Comment (Ailrun) [reply plugins/hls-brittany-plugin/test/testdata/BrittanyCRLF.formatted_range.hs:1]: Huh, it looks like this file is already in the ignored list... You still cannot make a commit without this change?

Comment (lunaticare) [reply plugins/hls-brittany-plugin/test/testdata/BrittanyCRLF.formatted_range.hs:1]: Will fix it, thank you for noticing!

Comment (lunaticare) [reply plugins/hls-brittany-plugin/test/testdata/BrittanyCRLF.formatted_range.hs]: Fixed!

**Review by Ailrun — APPROVED** [review]

**Pull request merged**